swagger_server/config.ini.template

```diff
@@ -5,3 +5,15 @@
 
 EVALUATION_TYPE=dryrun
 SCENARIO_DIRECTORY=swagger_server/itm/data/%(EVALUATION_TYPE)s/scenarios/
+
+ADEPT_PORT=8081
+ADEPT_HOSTNAME=localhost
+
+SOARTECH_PORT=8084
+SOARTECH_HOSTNAME=localhost
+
+SAVE_HISTORY=False
+HISTORY_DIRECTORY=itm_history_output
+
+SAVE_HISTORY_TO_S3=False
```

Comment:
As we discussed in Teams, we'll also make changes so we can point our local servers to the AWS TA1 servers (so we don't have to run them locally) but still have things work on prod.  The template will contain prod defaults.

So that means we need:
SOARTECH_URL=http://10.216.38.125:8084
ADEPT_URL=http://10.216.38.70:8080
SAVE_HISTORY=True
SAVE_HISTORY_TO_S3=True

The calls in `ITMTa1Controller` need to be updated appropriately, (e.g., `f"{url}/api/v1/xyz"`).

There should be a comment in the config template for the local URLs, e.g., ADEPT_URL="http://localhost:8081" identifying it as the local default.  Same for SoarTech (but 8084).

There should also be commented out versions for running locally, but against the AWS TA1 servers, e.g., ADEPT_URL="https://darpaitm.caci.com/adept" and SOARTECH_URL="https://darpaitm.caci.com/soartech"

And finally, based on our _other_ Teams conversation, could you remove the code override of `self.save_history_to_s3` and `self.save_history` in `itm_session.py`?

Whew!